src/Engine/Component/RigidBody.cpp

```diff
@@ -40,25 +45,49 @@ namespace Component {
     }
 
     glm::vec3 RigidBody::GetPosition() const {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetPosition' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)